Analysis/DataModel/include/AnalysisDataModel/HFSecondaryVertex.h

```diff
@@ -382,6 +386,38 @@ auto InvMassLcpiKp(const T& candidate)
 {
   return candidate.m(array{RecoDecay::getMassPDG(kPiPlus), RecoDecay::getMassPDG(kKPlus), RecoDecay::getMassPDG(kProton)});
 }
+
+// Ξc± → p± K∓ π±
+
+template <typename T>
+auto CtXic(const T& candidate)
+{
+  return candidate.ct(RecoDecay::getMassPDG(4232));
+}
+
+template <typename T>
+auto YXic(const T& candidate)
+{
+  return candidate.y(RecoDecay::getMassPDG(4232));
+}
+
+template <typename T>
+auto EXic(const T& candidate)
+{
+  return candidate.e(RecoDecay::getMassPDG(4232));
+}
+
+template <typename T>
+auto InvMassXicpKpi(const T& candidate)
+{
+  return InvMassLcpKpi(candidate);
+}
+
+template <typename T>
+auto InvMassXicpiKp(const T& candidate)
+{
+  return InvMassLcpiKpi(candidate);
+}
```

Comment:
@mfaggin The `InvMassLcpiKpi` function does not exist. Does O2 compile for you with this typo?

Reply:
@vkucera indeed it does not, sorry for the typo, I fixed it